packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -427,19 +429,51 @@ describe('common/sqlite', () => {
     expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
 
-                                                            - Introspecting based on datasource defined in prisma/invalid.prisma
+                                                                        - Introspecting based on datasource defined in prisma/invalid.prisma
 
-                                                            ✔ Introspected 3 models and wrote them into prisma/invalid.prisma in XXXms
-                                                                  
-                                                            Run prisma generate to generate Prisma Client.
+                                                                        ✔ Introspected 3 models and wrote them into prisma/invalid.prisma in XXXms
+                                                                              
+                                                                        Run prisma generate to generate Prisma Client.
 
-                                        `)
+                                                `)
```

Comment:
OT: One day I hope we will figure out what adds these useless spaces all the time and can fix it.

Reply:
I agree. Using jest with the `-u` flag causes this, from time to time. Not sure why exactly it happens, though